docker/volumes/mongo-fca-low/scripts/db-states/_default/sp.js

```diff
@@ -7,6 +7,7 @@ const fsa = {
     site: 'https://site.com',
     redirect_uris: [
       'https://fsa1-low.docker.dev-franceconnect.fr/oidc-callback',
+      "http://localhost:3000/login-callback"
```

Comment:
question: I do not understand this change. Maybe it was mistakenly added ?

Reply:
It was indeed mistakenly added!